src/styles/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  font-family: inherit;
```

Comment:
you don't need it, it's inherited by default
![Screenshot 2023-02-27 at 11 33 46](https://user-images.githubusercontent.com/75496546/221526950-9835262e-62fd-48a7-ae99-418728375f62.png)